cves/CVE-2011-3192.yml

```diff
@@ -3,22 +3,22 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: 400
 CVSS: AV:N/AC:L/Au:N/C:N/I:N/A:C
 curated_instructions: |
   If you are manually editing this file, then you are "curating" it. Set the
   entry below to "true" as soon as you start. This will enable additional
   integrity checks on this file to make sure you fill everything out properly.
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
-curated: false
+curated: true
 announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE date. A good
   source for this is Chrome's Stable Release Channel
   (https://chromereleases.googleblog.com/).
   Please enter your date in YYYY-MM-DD format.
-announced: 2011-08-29T15:55Z
+announced: 2011-08-29
```

Comment:
Is this the right date? SecurityFocus.com appears to have a post predating this by 5 days: https://www.securityfocus.com/bid/49303

Reply:
I think I had this originally, but changed it to reflect a press release from Apache. Fixed